tests/test_pyxis.py

```diff
@@ -405,56 +405,30 @@ def test_get_operator_indices_with_unreleased_filtered_out(self, page):
         assert "4.8" not in [i["ocp_version"] for i in indices]
 
     @patch('freshmaker.pyxis.Pyxis._pagination')
-    def test_get_bundles_per_index_image(self, page):
+    def test_get_latest_bundles(self, page):
+        self.maxDiff = None
         page_copy = self.copy_call_args(page)
-        page_copy.side_effect = [self.bundles[0:2], self.bundles[2:]]
-        out = self.px._get_bundles_per_index_image(self.indices)
+        page_copy.side_effect = [[self.bundles[2]], []]
 
-        expected_out = {"path/to/registry:v4.5": self.bundles[0:2],
-                        "path/to/registry:v4.6": self.bundles[2:]}
+        out = self.px.get_latest_bundles(self.indices)
+        expected_out = [self.bundles[2]]
 
         self.assertEqual(out, expected_out)
         page.assert_has_calls([
```

Comment:
There was a mistake in original code, so it should call `assert_has_call` method on `page_copy` not `page`. So then you might want to adjust call object below too.

Reply:
Fixed.